src/staker_log.cairo

```diff
@@ -0,0 +1,171 @@
+use starknet::storage::MutableVecTrait;
+use starknet::{get_block_timestamp};
+use starknet::storage_access::{StorePacking};
+
+use starknet::storage::{
+    Vec, VecTrait
+};
+use starknet::storage::{
+    StorageBase, Mutable, StorageAsPath, 
+    StoragePointerReadAccess, StoragePointerWriteAccess
+};
+
+pub type StakingLog = Vec<StakingLogRecord>;
+
+const TWO_POW_32: u64 = 0x100000000_u64;
+const MASK_32_BITS: u128 = 0x100000000_u128 - 1;
+const TWO_POW_160: u256 = 0x10000000000000000000000000000000000000000;
+pub const MAX_FP: u128 = 0x8000000000000110000000000000000_u128;
```

Comment:
what is this value?

Reply:
maximum value that can be store in felt252 is MAX_V = 0x800000000000011000000000000000000000000000000000000000000000000. 
MAX_FP = MAX_V >> 128 (high 123 bit).